src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -40,7 +40,11 @@ public ServiceManagerBuilder WithCallingAssembly()
         public IServiceManager Build()
         {
             _options.ValidateOptions();
-            return new ServiceManager(_options, ProductInfo.GetProductInfo(_assembly));
+
+            var productInfo = ProductInfo.GetProductInfo(_assembly);
+            ISignalRServiceRestClient restClient = new RestClientBuilder(_options.ConnectionString).WithUserAgent(productInfo).Build();
```

Comment:
Is it possible that the production info changes? can we input it in ctor of `RestClientBuilder`?